cinema/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class Movie(models.Model):
+
+    title = models.CharField(max_length=255)
+    description = models.CharField(max_length=255, null=True)
```

Comment:
The description could be a `TextField` without limitations for character numbers.